README.rst

```diff
@@ -131,8 +131,10 @@ Authors
 `Daniel Sank <https://github.com/DanielSank>`__ (Google),
 `Nicolas Sawaya <https://github.com/nicolassawaya>`__ (Harvard),
 `Kanav Setia <https://github.com/kanavsetia>`__ (Dartmouth),
-`Hannah Sim <https://github.com/hsim13372>`__ (Harvard) and
-`Wei Sun <https://github.com/Spaceenter>`__ (Google).
+`Hannah Sim <https://github.com/hsim13372>`__ (Harvard),
+`Wei Sun <https://github.com/Spaceenter>`__ (Google),
+`Chengyu Dai <https://github.com/jdaaph>`__ (University of Michigan) and
```

Comment:
This part of the author list is alphabetical by last name. So please put your names in the correct order.

Reply:
You should also update the NOTICE file. The NOTICE file is required by law to stay with any forks of the repo so it will make sure you get credit forever!